airflow/api_fastapi/parameters.py

```diff
@@ -148,17 +150,29 @@ def depends(self, dag_display_name_pattern: str | None = None) -> _DagDisplayNam
         return self.set_value(dag_display_name_pattern)
 
 
+# SortParam Implementations
 class SortParam(BaseParam[str]):
     """Order result by the attribute."""
 
     attr_mapping = {
         "last_run_state": DagRun.state,
         "last_run_start_date": DagRun.start_date,
+        "connection_id": Connection.conn_id,
+        "conn_type": Connection.conn_type,
+        "description": Connection.description,
+        "host": Connection.host,
+        "port": Connection.port,
+        "id": Connection.id,
```

Comment:
Most of them you don't need. Thinks will fallback to the `self.model.column` automatically. 

```
getattr(
            self.model, lstriped_orderby
        )
```

Whe only need to use the `mapping` if the attribute is on a related / joined table, and not the default `Model` one.

Reply:
Agree. Deleted the additional ones :)